src/plans/details/tabs/Details/components/DetailsSection/components/PlanWarm/WarmDetailsItem.tsx

```diff
@@ -17,11 +16,11 @@ const WarmDetailsItem: FC<EditableDetailsItemProps> = ({ canPatch, plan, shouldR
 
   if (!shouldRender) return null;
 
-  const isWarm = getPlanIsWarm(plan);
+  const migrationType = getPlanMigrationType(plan);
   return (
     <DetailsItem
       title={t('Migration type')}
-      content={<PlanWarmLabel isWarm={isWarm} />}
+      content={<PlanMigrationTypeLabel migrationType={migrationType} />}
       helpContent={t('Whether this is a warm migration.')}
       crumbs={['spec', 'warm']}
```

Comment:
should this change as well?
If you change the detail item content, please adjust the entire component to fit the content, component name crumbs, text inside help icon (helpContent), and is the edit plan warm still fits here?

Reply:
Good catch. I added a new component for the live migration case so that it can be toggled on and off as well. (it might make sense at some point in the future to have one component that allows switching, but I'm not sure what it should look like. For now a toggle will work because only OpenShift supports live and it doesn't support warm.)